### ISSUE TITLE
将多个query参数聚合为一个POJO类，同时使用POJO和@RequestHeader会导致NullPointerException异常

### ISSUE THREAD

**Issue by Caixinyang** [apache/servicecomb-java-chassis#1613]
Caused by: java.lang.NullPointerException: null
	at org.apache.servicecomb.swagger.invocation.arguments.producer.ProducerArgumentsMapperFactory.generateDefaultParamMapper(ProducerArgumentsMapperFactory.java:117)
	at org.apache.servicecomb.swagger.invocation.arguments.producer.ProducerArgumentsMapperFactory.generateParamMapperByName(ProducerArgumentsMapperFactory.java:103)
	at org.apache.servicecomb.swagger.invocation.arguments.producer.ProducerArgumentsMapperFactory.collectSwaggerArgumentsMapper(ProducerArgumentsMapperFactory.java:96)
	at org.apache.servicecomb.swagger.invocation.arguments.ArgumentsMapperFactory.collectArgumentsMapper(ArgumentsMapperFactory.java:90)
	at org.apache.servicecomb.swagger.invocation.arguments.producer.ProducerArgumentsMapperFactory.createArgumentsMapper(ProducerArgumentsMapperFactory.java:65)
	at org.apache.servicecomb.swagger.engine.SwaggerEnvironment.createProducer(SwaggerEnvironment.java:187)
	at org.apache.servicecomb.core.definition.schema.ProducerSchemaFactory.getOrCreateProducerSchema(ProducerSchemaFactory.java:78)
	at org.apache.servicecomb.provider.rest.common.RestProducerProvider.init(RestProducerProvider.java:45)
	at org.apache.servicecomb.core.provider.producer.ProducerProviderManager.init(ProducerProviderManager.java:54)
	at org.apache.servicecomb.core.SCBEngine.doInit(SCBEngine.java:246)
	at org.apache.servicecomb.core.SCBEngine.init(SCBEngine.java:214)
	... 12 common frames omitted

接口定义如下：
@RequestMapping(value = "/events", method = RequestMethod.GET)
    public ResponseEntity<?> queryEvents(@PathVariable(value = "project_id") String projectId,
        @RequestHeader(value = TOKEN_KEY) String token, @Valid QueryAlarmEventBody body) {
        return null;
    }

**Comment by heyile** [comment]
最好给下demo. 至少给下 SDK 版本信息

**Comment by Caixinyang** [comment]
<dependency>
            <groupId>com.huawei.paas.cse</groupId>
            <artifactId>cse-solution-service-engine</artifactId>
           <version>2.3.70</version>
            </exclusions>
        </dependency>

**Comment by heyile** [comment]
确实有问题. 问题我复现了 提了 issue https://issues.apache.org/jira/projects/SCB/issues/SCB-1793

**Comment by heyile** [comment]
你可以使用 如下进行规避

```
  @RequestMapping(value = "/events2/{project_id}", method = RequestMethod.GET)
  public ResponseEntity<Object> queryEvents2(@PathVariable(value = "project_id") String projectId,
          @RequestHeader(name = "x-auth22") Integer token2,
          @RequestHeader(name = "x-auth") Integer token, Person person) {
    return null;
  }

```

**Comment by heyile** [comment]
当前对value 的实现有问题. 是按照name 实现的

**Comment by liubao68** [comment]
fixed in 2.0.1
